packages/core/src/Cardano/types/Certificate.ts

```diff
@@ -40,7 +40,7 @@ export interface NewStakeAddressCertificate {
 export interface VoteDelegationCertificate {
   __typename: CertificateType.VoteDelegation;
   stakeKeyHash: Crypto.Ed25519KeyHashHex;
-  drep: DelegateRepresentative;
+  dRep: DelegateRepresentative;
```

Comment:
This and some other type changes in `Governance.ts` are technically breaking, please reflect it in commit message

Reply:
If we intend to squash this commit to the one that created Governance.ts then yeah

Reply:
Could also just cherry-pick this commit onto the other branch then squash it when ready to merge. I agree it's part of the same atomic change